todocore/src/main/java/com/knighttodo/todocore/service/privatedb/representation/Day.java

```diff
@@ -1,19 +1,15 @@
 package com.knighttodo.todocore.service.privatedb.representation;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.OneToMany;
-import javax.persistence.Table;
+import lombok.*;
```

Comment:
remove wildcards please and try not to update imports a lot, as smaller your PR is, is easier to other people to review it

Reply:
Done!